modules/custom-css/custom-css-4.7.php

```diff
@@ -0,0 +1,498 @@
+			$csstidy->set_cfg( 'remove_last_;',              false );
+			$csstidy->set_cfg( 'case_properties',            false );
+			$csstidy->set_cfg( 'discard_invalid_properties', true );
+			$csstidy->set_cfg( 'css_level',                  'CSS3.0' );
+			$csstidy->set_cfg( 'preserve_css',               true );
+			$csstidy->set_cfg( 'template',                   dirname( __FILE__ ) . '/csstidy/wordpress-standard.tpl' );
+
+			// Test for some preg_replace stuff.
+			{
+				$prev = $css;
+				$css = preg_replace( '/\\\\([0-9a-fA-F]{4})/', '\\\\\\\\$1', $css );
+				// prevent content: '\3434' from turning into '\\3434'
+				$css = str_replace( array( '\'\\\\', '"\\\\' ), array( '\'\\', '"\\' ), $css );
+				if ( $css !== $prev ) {
+					$warnings[] = 'preg_replace found stuff';
+				}
+			}
+
+			// Some people put weird stuff in their CSS, KSES tends to be greedy
+			$css = str_replace( '<=', '&lt;=', $css );
+
+			// Test for some kses stuff.
+			{
+				$prev = $css;
+				// Why KSES instead of strip_tags?  Who knows?
+				$css = wp_kses_split( $css, array(), array() );
+				$css = str_replace( '&gt;', '>', $css ); // kses replaces lone '>' with &gt;
+				// Why both KSES and strip_tags?  Because we just added some '>'.
+				$css = strip_tags( $css );
+
+				if ( $css != $prev ) {
+					$warnings[] = 'kses found stuff';
+				}
+			}
+
+			// if we're not using a preprocessor
+			if ( ! $args['preprocessor'] ) {
+
+				/**
+				 * Fires before parsing the css with CSSTidy, but only if
+				 * the preprocessor is not configured for use.
+				 *
+				 * @module custom-css
+				 *
+				 * @since 1.7.0
+				 *
+				 * @param obj $csstidy The csstidy object.
+				 * @param string $css Custom CSS.
+				 * @param array $args Array of custom CSS arguments.
+				 */
+				do_action( 'safecss_parse_pre', $csstidy, $css, $args );
+
+				$csstidy->parse( $css );
+
+				/**
+				 * Fires after parsing the css with CSSTidy, but only if
+				 * the preprocessor is not configured for use.
+				 *
+				 * @module custom-css
+				 *
+				 * @since 1.7.0
+				 *
+				 * @param obj $csstidy The csstidy object.
+				 * @param array $warnings Array of warnings.
+				 * @param array $args Array of custom CSS arguments.
+				 */
+				do_action( 'safecss_parse_post', $csstidy, $warnings, $args );
+
+				$css = $csstidy->print->plain();
+			}
+		}
+		return $css;
+	}
+
+	/**
+	 * Override $content_width in customizer previews.
+	 */
+	public static function preview_content_width() {
+		if ( isset( $GLOBALS['wp_customize'] ) ) {
+			if ( isset( $_POST['customized'] ) && isset( $_POST['customize_messenger_channel'] ) ) {
+				$customizations = json_decode( stripslashes( $_POST['customized'] ) );
+
+				if ( isset( $customizations->{'jetpack_custom_css[content_width]'} ) ) {
+					$width = (int) $customizations->{'jetpack_custom_css[content_width]'};
+
+					if ( $width ) {
+						$GLOBALS['content_width'] = $width;
+					}
+				}
+			}
```

Comment:
As noted below, the logic in this method should be changed to:

```php
global $wp_customize;
if ( ! is_customize_preview() ) {
    return;
}
$setting = $wp_customize->get_setting( 'jetpack_custom_css[content_width]' );
if ( ! $setting ) {
    return;
}
$customized_content_width = $setting->post_value();
if ( null !== $customized_content_width ) {
    $GLOBALS['content_width'] = $customized_content_width;
}
```

Note that the return value from the call to `$setting->post_value()` will have the setting's associated `sanitize_callback` applied.